tracing-subscriber/src/fmt/format/mod.rs

```diff
@@ -908,6 +908,17 @@ impl<T> Format<Json, T> {
         self.format.with_span_list(display_span_list);
         self
     }
+
+    /// Sets whether or not the formatter will include newline characters after
+    /// formatting a log event.
+    ///
+    /// See [`Json`]
```

Comment:
the same suggestions i left on the `Subscriber::with_newlines` method apply here as well